### PR TITLE
updating divider and cleanup markdownstring

### PR DESCRIPTION
**Pull request by tkimenyi**

### CODE REVIEW

File: src/player/index.ts [hunk 1]
```diff
@@ -233,7 +234,12 @@ function getNextTour(): CodeTour | undefined {
 
 function getHeaderText(step: CodeTourStep): string {
   if (isAccessibilitySupportOn() && step.line && step.file) {
```
Comment (m-giambrone) [inline src/player/index.ts]: The check on line 238 seems redundant with line 236? it looks like you might've meant to also update the if statement in what's now line 236?

File: src/player/index.ts [hunk 2]
```diff
@@ -55,8 +55,7 @@ const TOUR_REFERENCE_PATTERN =
   /(?:\[(?<linkTitle>[^\]]+)\])?\[(?=\s*[^\]\s])(?<tourTitle>[^\]#]+)?(?:#(?<stepNumber>\d+))?\](?!\()/gm;
 const FILE_REFERENCE_PATTERN = /(\!)?(\[[^\]]+\]\()(\.[^\)]+)(?=\))/gm;
 const CODE_FENCE_PATTERN = /```[^\n]+\n(.+)\n```/gms;
-
-const DIVIDER = "\n\n---\n";
+const DIVIDER = "\n";
```
Comment (m-giambrone) [inline src/player/index.ts]: I wonder if we could make the divider conditional on whether `isAccessibilitySupportOn()` is true - that way we'd be leaving the default behavior as is, but cleaning things up for the screenreader output?

File: src/player/index.ts [hunk 3]
```diff
@@ -133,6 +132,8 @@ export class CodeTourComment implements Comment {
 
     this.body = new MarkdownString(body);
     this.body.isTrusted = true;
+    this.body.supportHtml = true;
```
Comment (m-giambrone) [inline src/player/index.ts:135]: I'm super curious - do these props work like we expected from our meeting earlier this week? Or what's the before and after behavior for setting them?

File: package-lock.json [hunk 4]
```diff
@@ -1,12 +1,12 @@
 {
   "name": "codetour",
-  "version": "0.0.59",
+  "version": "0.0.60",
```
Comment (m-giambrone) [inline package-lock.json:3]: Were all of the changes here just the result of running `npm install` or did you have to specifically upgrade anything?

File: src/player/index.ts (hunk 3 above)
Comment (tkimenyi) [reply src/player/index.ts:135]: improves the view and it now reads the icons better so I guess they are working as expected. I don't think they improve screen reader experience as much though, but setting them is the right thing to do imo

File: package-lock.json (hunk 4 above)
Comment (tkimenyi) [reply package-lock.json:3]: result of running npm install, i did not install anything.

File: src/player/index.ts [hunk 5]
```diff
@@ -55,8 +55,7 @@ const TOUR_REFERENCE_PATTERN =
   /(?:\[(?<linkTitle>[^\]]+)\])?\[(?=\s*[^\]\s])(?<tourTitle>[^\]#]+)?(?:#(?<stepNumber>\d+))?\](?!\()/gm;
 const FILE_REFERENCE_PATTERN = /(\!)?(\[[^\]]+\]\()(\.[^\)]+)(?=\))/gm;
 const CODE_FENCE_PATTERN = /```[^\n]+\n(.+)\n```/gms;
-
-const DIVIDER = "\n\n---\n";
+const DIVIDER = isAccessibilitySupportOn() ? "\n" : "\n\n---\n";
```
Comment (brian-dot-net) [inline src/player/index.ts:58]: I feel it's better to do conditional logic to select the divider type, though I guess you'd have to change the pattern of the calling code a bit (pass another param...)

File: src/player/index.ts [hunk 6]
```diff
@@ -232,8 +233,13 @@ function getNextTour(): CodeTour | undefined {
...
 
 function getHeaderText(step: CodeTourStep): string {
-  if (isAccessibilitySupportOn() && step.line && step.file) {
-    const lineAndFileInfoLabel = `This step is on line ${step.line} in file ${step.file}.`;
+  if (isAccessibilitySupportOn()) {
+    let lineAndFileInfoLabel = "";
+    if (step.line && step.file) {
+      lineAndFileInfoLabel = `This step is on line ${step.line} in file ${step.file}.`;
+    } else {
+      lineAndFileInfoLabel = "There is no file associated with this step";
```
Comment (brian-dot-net) [inline src/player/index.ts]: In my experience, you need to put a period at the end for NVDA to properly pause at the end of the sentence.

File: src/player/index.ts (hunk 1 above)
Comment (m-giambrone) [reply src/player/index.ts]: resolving :)

File: src/player/index.ts (hunk 2 above)
Comment (m-giambrone) [reply src/player/index.ts]: awesome!

File: src/player/index.ts (hunk 3 above)
Comment (m-giambrone) [reply src/player/index.ts:135]: ok sounds reasonable, thanks for the details!

File: src/player/index.ts (hunk 6 above)
Comment (m-giambrone) [reply src/player/index.ts]: Yeah, explicit punctuation is usually the best way to ensure that there's a pause.

File: src/player/index.ts (hunk 5 above)
Comment (tkimenyi) [reply src/player/index.ts:58]: do you mean conditional logic in addition to checking `isAccessibilitySupportOn()`?

Comment (brian-dot-net) [reply src/player/index.ts:58]: I mean the condition should be in the function below, not in the const declaration. You could have two const dividers and then select the correct one just in time.

Comment (tkimenyi) [reply src/player/index.ts:58]: Oh I see. I think the code is cleaner and simpler this way. We can change it later if necessary.

**Pull request merged**